static/app/components/sidebar/index.tsx

```diff
@@ -276,6 +278,17 @@ function Sidebar() {
                   {...HTTPModuleBadgeProps}
                 />
               </Feature>
+              <Feature features="performance-cache-view" organization={organization}>
+                <SidebarItem
+                  {...sidebarItemProps}
+                  label={
+                    <GuideAnchor target="performance-cache">{t('Cache')}</GuideAnchor>
+                  }
+                  to={`/organizations/${organization.slug}/performance/cache/`}
+                  id="performance-cache"
+                  icon={<SubitemDot collapsed />}
```

Comment:
Missing `releaseLevelAsBadgeProps` maybe?

Reply:
Yup, you're right!